R/plot-build.r

```diff
@@ -261,6 +266,70 @@ ggplot_gtable.ggplot_built <- function(data) {
   plot_table <- gtable_add_grob(plot_table, caption, name = "caption",
     t = -1, b = -1, l = min(pans$l), r = max(pans$r), clip = "off")
 
+  plot_table <- gtable_add_rows(plot_table, unit(0, 'pt'), pos = 0)
```

Comment:
Why do we need all these new rows and cols?

Reply:
Should they always be added even when there isn't a tag? I don't have sense of if adding new rows/cols will disrupt existing code

Reply:
The idea here, which is formed both by my work with facets and my extensions, is that it is much nicer to have a predictable gtable layout to work with. This is also why rows and columns are added for axes and strips on all sides even if they aren’t drawn. If any external code depend on the gtable layout it will break either way when users start to add tags, but doing it this way makes it much easier to fix as you don’t need any gtable introspection to see if a tag is added, and where.

There is nothing inside ggplot2 that breaks so this is more a breaking change for extensions